src/plugins/Minecraft/controllers/events/minecraftOnGuildMemberUpdate.js

```diff
@@ -1,25 +1,26 @@
 const Discord = require("discord.js");
 const metrics = require("../../../../index.js");
-const db = require("quick.db");
+const dbh = require("../../../.common/structures/DataHandling/DatabaseHandler.js");
 
 exports.eventHandle = "guildMemberUpdate";
 exports.event = async (client, oldMember, newMember) => {
-  metrics.sendEvent("ready");
+  const { guild } = newMember;
 
   //If the user did not get a new role, ignore it.
   if (!(oldMember._roles.length < newMember._roles.length)) return;
+  const roleAdded = newMember._roles.filter((r) => !oldMember._roles.includes(r))[0];
+  const minecraftRole = await dbh.guild.getMinecraftRole(guild.id);
 
-  // prettier-ignore
-  let roleAdded = newMember._roles.filter((r) => !oldMember._roles.includes(r))[0];
-  let guild = newMember.guild;
-  if (roleAdded === db.get(`minecraft-role.${guild.id}`)) {
-    let mcChannelId = db.get(`minecraft-channel.${guild.id}`);
-    if (!mcChannelId);
-    let mcChannel = guild.channels.resolve(mcChannelId);
-    mcChannel.sendEmbed({
-      color: 0xff9f01,
-      description: `${newMember.toString()} Welcome to Minecraft!
-      Server IP: minecraft.devlaunchers.org`,
-    });
-  }
+  if (roleAdded !== minecraftRole) return;
+  const minecraftChannelId = await dbh.channels.getMinecraft(guild.id);
+
+  if (!minecraftChannelId) return;
+  const minecraftChannel = guild.channels.resolve(minecraftChannelId);
+  if (!minecraftChannel) return;
+
+  minecraftChannel.sendEmbed({
+    color: 0xff9f01,
+    description: `${newMember.toString()} Welcome to Minecraft!
+    Server IP: minecraft.devlaunchers.org`,
```

Comment:
This is the hostname, not IP.

Reply:
In many places, the refer to the hostname as the Server IP
![image](https://user-images.githubusercontent.com/55030854/164113597-7610418b-7e0f-4d28-80c2-95d6f30e5038.png) 
I changed it to Server Address which is more correct and is the name of the input box the users will enter `minecraft.devlaunchers.org` to access the server.